src/array_api_stubs/_draft/array_object.py

```diff
@@ -748,6 +757,9 @@ def __invert__(self: array, /) -> array:
 
         .. note::
            Element-wise results must equal the results returned by the equivalent element-wise function :func:`~array_api.bitwise_invert`.
+
+        .. note::
+           Comparisons of arrays without a corresponding promotable data type (see :ref:`type-promotion`) is undefined and thus implementation-dependent.
```

Comment:
```suggestion
```

I don't think we need this note for `invert`, as it is not a comparison operation.